itest/rust/src/register_tests/keyword_parameters_test.rs

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use godot::classes::IEditorExportPlugin;
+use godot::prelude::*;
```

Comment:
One thing I missed before: could you directly import the relevant symbols (not wildcard, not prelude)? That implicitly tests that symbols are in the modules we expect them to be.